.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+check-filenames =
+builtin = clear,rare,usage,informal
+skip = */.git,*/cmake-build-*,*/.idea,*/completions,*/presets,*/screenshots,*/tests,*/3rdparty,*/logo/ascii
+ignore-words-list = iterm,compiletime,intergrated,unknwn,pengwin,siduction,master,sur,doas
```

Comment:
Fixing `integrated` should be ok. Just remember to mention it in the change log

Reply:
I removed it and added a changelog entry